sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Authorization/EventHubTokenCredential.cs

```diff
@@ -18,46 +18,33 @@ internal class EventHubTokenCredential : TokenCredential
         /// <summary>The default scope used for token authentication with EventHubs.</summary>
         private const string DefaultScope = "https://eventhubs.azure.net/.default";
 
-        /// <summary>
-        ///   The Event Hubs resource to which the token is intended to serve as authorization.
-        /// </summary>
-        ///
-        public string Resource { get; }
+        /// <summary>The <see cref="TokenCredential" /> that forms the basis of this security token.</summary>
+        private readonly TokenCredential _credential;
 
         /// <summary>
         ///   Indicates whether the credential is based on an Event Hubs
-        ///   shared access signature.
+        ///   shared access policy.
```

Comment:
nit?
```suggestion
        ///   shared access credential.
```

Reply:
For the messaging services, this is represented as a "shared access policy" in the portal, so I'm using that terminology here.

![image](https://user-images.githubusercontent.com/913445/112506458-290e9680-8d64-11eb-97b3-adf49d2c11a7.png)